app/views/users/profile.html.erb

```diff
@@ -154,14 +154,18 @@
 
   <hr />
   <div class="row" id="highlight" style=" text-align: center;">
-    <h3 class="col-md-4 col-xs-12 col-sm-4"><%= Node.questions.where(status: 1, uid: @user.id).length %> questions asked</h3>
-    <h3 class="col-md-4 col-xs-12 col-sm-4"><%= Answer.where(uid: @profile_user.id).count %> answers posted</h3>
+    <h3 class="col-md-4 col-xs-12 col-sm-4"><%= Node.questions.where(status: 1, uid: @user.id).length %> questions asked <br><br> <%= Answer.where(uid: @profile_user.id).count %> answers posted</h3>
     <h3 class="col-md-4 col-xs-12 col-sm-4"><%= Comment.where(uid: @profile_user.id).count %> comments posted</h3>
+     <% if !@map_lat.nil? && !@map_lon.nil? %>
+      <%= render :partial => "map/userLeaflet" , locals: {flag: true} %>
```

Comment:
Here, perhaps we could be more descriptive of what `flag` means -- maybe `hasLocation`?

Reply:
Okay :)